### ISSUE TITLE
Disable YouTube

### ISSUE THREAD

**Issue by adapana** [yt-dlp/yt-dlp#4160]
### Checklist

- [X] I'm asking a question and **not** reporting a bug or requesting a feature
- [X] I've looked through the [README](https://github.com/yt-dlp/yt-dlp#readme)
- [X] I've verified that I'm running yt-dlp version **2022.06.22.1** ([update instructions](https://github.com/yt-dlp/yt-dlp#update)) or later (specify commit)
- [X] I've searched the [bugtracker](https://github.com/yt-dlp/yt-dlp/issues?q=) for similar questions including closed ones. DO NOT post duplicates
- [X] I've read the [guidelines for opening an issue](https://github.com/yt-dlp/yt-dlp/blob/master/CONTRIBUTING.md#opening-an-issue)

### Question

Is there any option to disable download YouTube video?

Some site use YouTube video embed frame in their site and if I use that site URL in yt-dlp then that YouTube video downloads. I want to disable this type of video download.

In short, If YouTube video URL found in webpage, then skip download.

If this is doable, then please guide me.

**Comment by flashdagger** [comment]
`--match-filter 'extractor!~=youtube'`

**Comment by adapana** [comment]
@flashdagger Thank you very much.

It is working as expected.
